anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-failover.adoc

```diff
@@ -0,0 +1,118 @@
+= Using Cross-Region Failover
+
+Anypoint MQ Connector for Mule 4 version 3.2.6 and later provides failover capability for standard queues.
+
+This feature is not enabled by default.
+Before using this feature, review xref:mq::mq-failover.adoc[].
+
+[IMPORTANT]
+When you enable failover for a queue and the primary region goes down,
+your data resides in a different region.
```

Comment:
should we clarify that the data is routed to a different region only until the primary region comes back up?